dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -240,119 +243,122 @@ public final Response containerContent(@Context final HttpServletRequest req, @C
     public final Response removeContentletFromContainer(@Context final HttpServletRequest req,
             @Context final HttpServletResponse res, @PathParam("containerId") final String containerId,
             @PathParam("contentletId") final String contentletId, @QueryParam("order") final long order,
-            @PathParam("uid") final String uid) throws DotDataException, DotSecurityException, ParseErrorException,
+            @PathParam("uid") final String uid) throws DotDataException, ParseErrorException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.apache.velocity.exception.ParseErrorException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)